composer.json

```diff
@@ -26,7 +26,8 @@
         "symfony/http-kernel": "^2.3 || ^3.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
-        "symfony/routing": "^2.3 || ^3.0"
+        "symfony/routing": "^2.3 || ^3.0",
+        "phpoffice/phpexcel": "1.8.*"
```

Comment:
Is it better for all versions of MS Office? Anyway, changing the default xls writer would be a BC-break IMO, which mean you wouldn't be able to make the PR on this branch. I think we should make it optional in this PR, and change it later on the master branch if it turns out to be indeed better.